src/collections/_documentation/api/permissions.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Permissions
+sidebar_order: 3
+---
+
+If you're building on top of Sentry's API (i.e using [Auth Tokens]({%- link _documentation/api/auth.md -%})), you'll need certain scopes to access
+different API endpoints.
+
+If you're looking for information on membership roles please visit the
+[membership]({%- link _documentation/accounts/membership.md -%}) documentation.
+
+**Organizations***
+
+```bash
+GET      'org:read'    
+PUT/POST 'org:write'
+DELETE   'org:admin'
+```
+
+**Projects***
+```bash
+GET      'project:read'    
+PUT/POST 'project:write'
+DELETE   'project:admin'
+```
+\* The `project:releases` scope will give you access to both **project**
+and **organization** release endpoints. The available endpoints are listed in
+the [Releases]({%- link _documentation/api/releases/index.md -%}) section of the API Documentation.
+
+**Teams**
+```bash
+GET      'team:read'    
+PUT/POST 'team:write'
+DELETE   'team:admin'
+```
+
+**Members**
+```bash
+GET      'member:read'    
+PUT/POST 'member:write'
+DELETE   'member:admin'
+```
+
+**Issues & Events**
+
+Please note that PUT/DELETE methods only apply to updating/deleting issues.
+Events in sentry are immutable and can only be deleted by deleting the whole issue.
```

Comment:
Since there is a conditional difference between them, why not make Issues and Events separate entries in the list?

Reply:
I figured because they are so tightly related (especially in the UI) that they should go together. I think the confusing part is that our scope is `event:write` which seems to imply that you can edit an event so I would think that this 'disclaimer' would still be useful anyhoo